docs/components/ColorBlock.tsx

```diff
@@ -1,23 +1,64 @@
 import { clsx } from "clsx";
+import { useEffect, useState } from "react";
+import {
+  characteristic,
+  getCharacteristicValue,
+  makePrefixer,
+} from "@salt-ds/core";
 
 import "./ColorBlock.css";
 
+const withBaseName = makePrefixer("ColorBlock");
+
 export const ColorBlock = ({
   colorVar,
   className,
+  hideToken,
 }: {
   colorVar: string;
   className?: string;
+  hideToken?: boolean;
 }) => {
+  const characteristicName = colorVar
+    .split("--salt-")[1]
+    .split("-")[0] as characteristic;
+  const [color, setColor] = useState<string | null>();
+  const [withBorder, setWithBorder] = useState<boolean>(false);
+  const [transparent, setTransparent] = useState<boolean>(false);
+
+  useEffect(() => {
+    setColor(
+      getCharacteristicValue(
+        "salt-theme",
+        characteristicName,
+        colorVar.split(`${characteristicName}-`)[1]
+      )
+    );
+  }, []);
```

Comment:
shouldn't this have colorVar as a dep? and maybe move characteristicName into useEffect?

Reply:
colorVar won't change so I don't think it's necessary to add as a dep?